samples/apps/copilot-chat-app/webapi/DeploymentTemplates/README.md

```diff
@@ -125,6 +125,20 @@ check that you have correctly entered the values for the following settings:
 Both Completion:Endpoint and Embedding:Endpoint are ignored for OpenAI instances from [openai.com](https://openai.com) but MUST be properly populated when using Azure OpenAI instances.
 
 
+## Authorization
+
+All of the server's endpoints other than the /probe one require authorization to access.
+By default, the deployment templates set up the server so that an API key is required to access its endpoints.
+
+AAD authentication and authorization can also be set up manually after the automated deployment is done.
+
+To view the API key required by your instance, access the page for your Semantic Kernel app service in the Azure portal.
+From that page, click on the "Configuration" item in the "Settings" section of the left pane. Then click on the text that reads "Hidden value.
+Click to show value" next to the "Authorization:ApiKey" setting.
+
+To authorize requests with the API key, it must be added as the value of an "x-api-key" header added to the requests.
```

Comment:
I don't see any SK app service logic changes that check the "x-api-key" header. Do we plan to have another PR to cover it?

Reply:
Correct.